packages/sampleCommons/src/pages/_app.tsx

```diff
@@ -65,8 +59,8 @@ const Gen3App = ({
     //   process.env.NEXT_PUBLIC_FARO_APP_ENVIRONMENT != "local" &&
     //   !faroRef.current
     // ) {
-    // Note: not using faro for development
-    // if (!faroRef.current) faroRef.current = initGrafanaFaro();
+
+    if (!faroRef.current) faroRef.current = initGrafanaFaro();
```

Comment:
The only problem with this is if we do a lot of development, then this will add "noise" to the graphana logs.
How about adding the check in the commented-out code: `process.env.NEXT_PUBLIC_FARO_APP_ENVIRONMENT != "local" `and remove the commented-out code?

Reply:
thanks for reviewing i was trying to just bring in the changes from [commons-frontend-app](https://github.com/ocshawn/commons-frontend-app) in retrospect i don't know which version of the code is more up to date, i was assuming commons-frontend-app